Xamarin.Forms.NeoControls/NeoView.xaml.cs

```diff
@@ -80,13 +87,25 @@ public Color DarkShadowColor
             set => SetValue(DarkShadowColorProperty, value);
         }
 
+        public Color BaseColor
```

Comment:
você comentou que precisou alterar para uma cor base, mas eu não entendi direito o porque.
Caso queira fazer alguma diferenciação da cor de fundo, não seria apenas de escolher alguma cor diferente para a mascara? uma nova propriedade de cor aplicada em cima do background?
Falo isso pois quem for usar esses controles provavelmente vai tentar aplicar o background para tentar alterar a cor total do fundo do controle.

Reply:
Ele pode fazer isso e colocar uma cor no BackgroundColor, mas uma coisa é o fundo e outra  é a cor do objeto 3d, se por acaso ele quiser colocar sobre um fundo degrade ou textura de photo, ele precisa estar transparente ( Color.Transparente ).

Reply:
<img width="241" alt="image" src="https://user-images.githubusercontent.com/1719565/80488464-4a99af00-8956-11ea-8710-99713a1ed80d.png">

Se colocar o fundo essa imagem acima não será possível

Reply:
humm, interessante...
O botão utilizando o backgroundcolor consegui inserir a imagem por cima..
Vou verificar se encontro uma alternativa pra essa situação, pois como os demais controles ja foram baseados no BackgroundColor, nao seria muito bom mudar a classe base.

Reply:
Uma solução seria armazenar o valor do BackgroundColor, colocar ele Transparent, e usar esse hack _BackgroundColor ou BaseColor, para substituir, dessa forma no Xaml continua com BackgroundColor

Reply:
Isso teria que ser feito antes da criação do canvas